monai/networks/nets/hovernet.py

```diff
@@ -424,7 +418,7 @@ class HoVerNet(nn.Module):
             get the same output size as the input, and this changed version is used on CoNIC challenge.
             Please note that to get consistent output size, `HoVerNetMode.FAST` mode should be employed.
         dropout_prob: dropout rate after each dense layer.
-        pretrained: whether to load pretrained weights for encoder.
+        pretrained_url: if specifying, will loaded the pretrained weights of preact resnet50 downloaded from the url.
```

Comment:
Each user is responsible for checking the content of model/datasets and the applicable licenses and determining if suitable for the intended use.